charts/pgadmin4/templates/deployment.yaml

```diff
@@ -29,14 +29,23 @@ spec:
         {{- with .Values.podLabels }}
         {{- toYaml . | nindent 8 }}
         {{- end }}
-    {{- if or (not .Values.existingSecret) .Values.podAnnotations }}
+    {{- if or (not .Values.existingSecret) .Values.podAnnotations .Values.templatedPodAnnotations .Values.serverDefinitions.enabled }}
```

Comment:
I am not certain if this conditional block works, it now needs those 3 values to meet the condition?